src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,378 @@
+                        "type": "string",
+                        "enum": [
+                            "GRCh38",
+                            "GRCm39"
+                        ]
+                    },
+                    "chromosome": {
+                        "title": "Chromosome",
+                        "description": "The number (or letter) designation for the chromosome, e.g. chr1 or chrX",
+                        "type": "string",
+                        "pattern": "^(chr[0-9A-Za-z]+)$"
+                    },
+                    "start": {
+                        "title": "Start",
+                        "description": "The 1-based starting coordinate.",
+                        "type": "integer",
+                        "minimum": 0
+                    },
+                    "end": {
+                        "title": "End",
+                        "description": "The 1-based ending coordinate.",
+                        "type": "integer",
+                        "minimum": 0
+                    }
+                }
+            }
+        },
+        "plasmid_map": {
+            "title": "Plasmid Map",
+            "description": "An image of the plasmid map used in this construct library.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "comment": "See document.json for a list of available identifiers.",
+                "type": "string",
+                "linkTo": "Document"
+            }
+        },
+        "average_size": {
+            "title": "Average Size",
+            "description": "The average size of the inserts (such as guides) cloned into vectors in the library.",
+            "comment": "May not be submitted with size_range.",
+            "type": "number",
+            "minimum": 0
+        },
+        "size_range": {
+            "title": "Size Range",
+            "description": "The numerical range of the size of the inserts (such as guides) cloned in vectors in the library.",
+            "comment": "May not be submitted with average_size.",
+            "type": "string",
+            "pattern": "(^[0-9]+-[0-9]+$|^[<>][0-9]+$)"
+        },
+        "mpra_library_detail": {
+            "title": "MPRA Construct Library Detail",
+            "description": "Details about an MPRA, STARR-seq, or other reporter assay construct library.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "MPRA Library",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "complexity": {
+                        "title": "Complexity",
+                        "description": "The complexity (number of constructs) in the library.",
+                        "type": "integer",
+                        "minimum": 0
+                    }
```

Comment:
aren't you already asking for quantity? how is that different from complexity?

Reply:
Described in detail on the ticket, now broken to mpra_library_complexity and average_guide_coverage/guide_coverage_range

Reply:
Have now removed this as discussed (lack a consistent definition between different labs doing MPRA)